### DISCUSSION TITLE
求教安装nb_cli后出现这个问题怎么解决

### DISCUSSION THREAD

**Discussion by Darkerami** [nonebot/discussions#314] [问答]
礼貌求教求教
nb create
[?] 选择一个要使用的模板: simple (开发者)
正在加载适配器...
ModuleLoadFailed('获取 adapter 列表失败.', [ConnectError('EOF occurred in violation of protocol (_ssl.c:1129)'), ConnectError('EOF
 occurred in violation of protocol (_ssl.c:1129)'), ConnectError('EOF occurred in violation of protocol (_ssl.c:1129)')])

**Comment by StarHeartHunt** [comment]
请确保你运行的机器的网络环境正常，正确配置了网络代理。
之后尝试 `netsh winsock reset`。

**Reply by Darkerami** [reply]
还有这个安装错误咋办呀？
![image](https://user-images.githubusercontent.com/118607168/217140100-fd181582-0bd4-4d10-974a-dbb3831802ee.png)
